.gitmodules

```diff
@@ -1,9 +1,12 @@
 [submodule "loader"]
 	path = external/OpenCL-ICD-Loader
-	url = https://github.com/KhronosGroup/OpenCL-ICD-Loader.git
+	url = git@projects.streamhpc.com:Khronos/OpenCL-ICD-Loader.git
```

Comment:
The paths currently point to https://github.com/StreamHPC/... because they all need CMake changes. As the "SDK base changes" PRs are merged, they can be reverted to the KhronosGroup hosted repos.